spec/models/school_spec.rb

```diff
@@ -124,7 +124,7 @@
       end
 
       it 'is true' do
-        expect(school.has_laptop_allocation?).to eq(true)
+        expect(school.laptops_allocated?).to eq(true)
```

Comment:
Again with the old name you could have got the `have_laptop_allocation` RSpec matcher

Reply:
See my comment above